### DISCUSSION TITLE
2011/03/11/%E8%B1%86%E8%B1%86%E9%95%BF%E7%9A%84%E7%9C%9F%E5%BF%AB%E5%95%8A

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#317] [Announcements]
# 2011/03/11/%E8%B1%86%E8%B1%86%E9%95%BF%E7%9A%84%E7%9C%9F%E5%BF%AB%E5%95%8A

豆豆今天去医院打疫苗，顺便量了身高体重：身高69cm，体重7.7公斤。

https://blog.qizhen.xyz/2011/03/11/%E8%B1%86%E8%B1%86%E9%95%BF%E7%9A%84%E7%9C%9F%E5%BF%AB%E5%95%8A

**Comment by ruanqizhen** [comment]
之前博客下的留言：

勺子
2011年03月11日 19:37 编辑
后面会稍慢一些些的啦，开始劲道都足的。

回复
